.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 .sass-cache
 .DS_Store
 .tern-project
+dist
```

Comment:
You'd also want to do:

git rm -rf --cached .
git add .
git commit -m "chore: remove dist files"